Microsoft.Toolkit.Uwp.UI.Controls.WinRT/pch.h

```diff
@@ -0,0 +1,15 @@
+﻿#pragma once
+
+#include <winrt/base.h>
+#include <vector>
+#include <memory>
+#include <algorithm>
+#include <string>
+#include <d3d11.h>
+#include <d2d1_1.h>
+#include <dwrite.h>
+
+#define BEGIN_NAMESPACE_CONTROLS_WINRT namespace Microsoft { namespace Toolkit { namespace Uwp { namespace UI { namespace Controls { namespace WinRT {
```

Comment:
remove WinRT from namespace, change root namespace to Microsoft.Toolkit.Uwp.UI.Controls